clang/test/Modules/pr73023.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: split-file %s %t
+//
+// RUN: %clang_cc1 -std=c++20 %t/foo.cpp -I%t -fsyntax-only -verify
```

Comment:
You don't really need `-I%t`, the test works without it. But don't have a strong opinion.

Reply:
Done.